sidebery.json

```diff
@@ -2,19 +2,61 @@
   "settings": {
     "nativeScrollbars": true,
     "nativeScrollbarsThin": true,
-    "ctxMenuNative": true,
-    "navBarLayout": "hidden",
+    "updateSidebarTitle": true,
+    "ctxMenuNative": false,
+    "navBarLayout": "horizontal",
+    "navBarInline": true,
     "subPanelRecentlyClosedBar": false,
     "subPanelBookmarks": false,
     "subPanelHistory": false,
-    "showNewTabBtns": true,
```

Comment:
I would probably leave the new tab button enabled by default, as that is what Edge displays.